spec/models/time_profile_spec.rb

```diff
@@ -11,6 +11,105 @@
     expect(t.tz).to be_nil
   end
 
+  describe "Test profile= with days" do
+    before do
+      @tp = TimeProfile.new
```

Comment:
Unless you explicitly don't want this record to be created, usually the way to do this is:
```
let(:time_profile) { FactoryBot.create(:time_profile) }

before { time_profile.profile = data }
```

Then just use `time_profile` everywhere instead of `@tp`